src/main/java/entities/Player.java

```diff
@@ -24,7 +26,9 @@ public void drawCards(List<Card> cards) {
         hand.addAll(cards);
     }
 
-    public void playCard(int n) {
+    public Card playCard(int n) {
+        Card toPlay = hand.get(n);
         hand.remove(n);
```

Comment:
I believe hand.remove(n) removes _and_ returns the card at index n. You can write this in line 29 in place of hand.get(n)

Reply:
I changed it, thanks.